src/store/creator/types.ts

```diff
@@ -13,9 +13,13 @@ export type CreatorItem = {
     shape: Rect;
 };
 
+export type CreatorStagedItem = Omit<CreatorItem, 'shape'> & {
```

Comment:
The manipulation of the types is very confusing. Should we separate `message` into its own reducer-level property rather than keeping it in `staged`? It seems to be complicating things quite a bit.